documentation.html

```diff
@@ -703,6 +707,212 @@ <h4>Out of Stock</h4>
+            class="code-example"
+            sandbox="allow-scripts allow-same-origin"
+          >
+          </iframe>
+          <p class="component-sub-heading-bold">Javascript</p>
+          <!--  Code Example  -->
+          <iframe
+            loading="lazy"
+            src="https://carbon.now.sh/embed?bg=rgba%28224%2C224%2C224%2C1%29&t=cobalt&wt=none&l=javascript&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=1x&wm=false&code=function%2520change%28input%29%2520%257B%250A%2520%2520input.setAttribute%28%2522value%2522%252C%2520input.value%29%253B%250A%257D"
+            class="code-example"
+            sandbox="allow-scripts allow-same-origin"
+          >
+          </iframe>
+
+          <p class="component-sub-heading-bold">
+            Example of Inputs with Validation
+          </p>
+          <p class="component-sub-heading">
+            We have 3 types of validation with inputs
+            <span class="highlight-text">error</span>,
+            <span class="highlight-text">warn</span>,
+            <span class="highlight-text">success</span>
+          </p>
+          <div
+            class="component-example flex-row flex-row-center-vertical flex-gap-1"
+          >
+            <!--  Success Box  -->
+            <div class="input-box">
+              <input
+                type="text"
+                class="input success"
+                value=""
+                onchange="change(this)"
+                required
+              />
+              <label class="input-label">
+                <span>*</span>
+                Success Box
+              </label>
+            </div>
+            <!--    -->
+
+            <!--  Warning Box  -->
+            <div class="input-box">
+              <input
+                type="text"
+                class="input warning"
+                value=""
+                onchange="change(this)"
```

Comment:
This works but I am not sure if you need it. We maintain state and use onChange kind of working in React but in Vanilla JS I think you don't really need it. Not sure about this.

Reply:
Actually this is what which is enabling it to float up.